source/deploy/standalone/proxy_ssl.md.erb

```diff
@@ -0,0 +1,247 @@
+<span class="prompt">$ </span>sudo apt-get install -y openssl
+<span class="prompt">$ </span>sudo a2enmod ssl &#35; only if using apache</pre>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Red Hat, CentOS, Fedora, Amazon Linux, Scientific Linux
+            </td>
+            <td>
+                <pre class="highlight"><span class="prompt">$ </span>sudo yum update -y
+<span class="prompt">$ </span>sudo yum install -y openssl
+<span class="prompt">$ </span>sudo yum install -y mod24_ssl &#35; only if using httpd24
+<span class="prompt">$ </span>sudo yum install -y mod_ssl &#35; only if using httpd</pre>
+
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Mac OS X
+            </td>
+            <td>
+                <pre class="highlight"><span class="prompt">$ </span>sudo softwareupdate -i -a
+<span class="prompt">$ </span>brew update
+<span class="prompt">$ </span>brew upgrade
+<span class="prompt">$ </span>brew install openssl
+<span class="prompt">$ </span>sudo sed -E -i.bak -e 's|#(LoadModule ssl_)|\1|' /etc/apache2/httpd.conf &#35; if using apache</pre>
+            </td>
+        </tr>
+</table>
+
+## Software versions used in this article:
+
+To check the versions of the packages you have installed you can use the following commands:
+
+<table class="table table-bordered table-striped">
+        <thead>
+            <th>
+                Package
+            </th>
+            <th>
+                Version
+            </th>
+            <th>
+                Version Command
+            </th>
+        </thead>
+        <tr>
+            <td>
+                openssl
+            </td>
+            <td>
+                1.0.2h
+            </td>
+            <td>
+                <pre class="highlight"><span class="prompt">$ </span>openssl version</pre>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                passenger
+            </td>
+            <td>
+                5.0.29
+            </td>
+            <td>
+                <pre class="highlight"><span class="prompt">$ </span>passenger -v</pre>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                nginx
+            </td>
+            <td>
+                1.10.1
+            </td>
+            <td >
+                <pre class="highlight"><span class="prompt">$ </span>nginx -v</pre>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                apache httpd
+            </td>
+            <td>
+                 2.4.18
+            </td>
+            <td>
+                <pre class="highlight"><span class="prompt">$ </span>httpd -v || apache2 -v</pre>
+            </td>
+        </tr>
+</table>
+
+## Obtain your certificate / key pair
+You can read up on self signing a certificate in [this article](<% url_for 'devssl' %>) or if this is for production use you can use let's encrypt or another CA to get a valid cert.
```

Comment:
the linked information about self signing a certificate should be easily identifiable (e.g. have its own heading so you can link directly to it), it isn't really atm.